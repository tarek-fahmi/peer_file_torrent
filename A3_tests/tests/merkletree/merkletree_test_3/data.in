Completed Component - Valid File
./bin/pkg_main ./valid_1.bpkg -min_hashes
./bin/pkg_main ./valid_2.bpkg -min_hashes