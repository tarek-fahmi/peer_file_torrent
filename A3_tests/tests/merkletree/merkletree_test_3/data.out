d617b42e1b9ca2781f2f7ad64e62fc3aff7cabe133a4449a27d759b8b8ec1db7
6f1df1c3bcd7f68a780d397ad6eb448a77a819d352d7ac0ea6c4b04dbfe3279c